README.md

```diff
@@ -66,6 +74,38 @@ menuentry 'CentOS Linux GNU/Linux, with tboot 1.9.4 and Linux 3.10.0-514.el7.x86
 ### END /etc/grub.d/20_linux_tboot ###
 ```
 
+## EL6
+
+1. Create a tboot grub entry.
+    1. The following entry is based off of vanilla centos 6.9
+    2. Your kernel parameters may vary, and you can add/subtract as needed
```

Comment:
You need to generate new policy when changing the kernel parameters, it's not _exactly_ as-needed.